src/index.js

```diff
@@ -63,7 +67,26 @@ export default {
                 browserOptions(builder, caps);
         }
 
-        const webDriver = await builder.build();
+        let maxTries = Number(process.env.SELENIUM_MAX_TRIES) || 1;
+
+        const retryInterval = Number(process.env.SELENIUM_RETRY_INTERVAL) || 5e3;
```

Comment:
5e3 = 5,000, but in the readme you said the default was 50,000
Also just curious why you chose 50 seconds? :)

Reply:
It's a typo. Thank you to point out.